test/deployframework/framework.go

```diff
@@ -196,3 +210,61 @@ func (df *DeployFramework) NewDeployerConfig(
 		MeteringConfig:           meteringConfig,
 	}, nil
 }
+
+// CreateCatalogSource is a deployframework method responsible
+// for instantiating a new CatalogSource that can be used
+// throughout individual Metering installations.
+func (df *DeployFramework) CreateCatalogSource(registryImage, meteringOperatorImage, reportingOperatorImage string) (string, string, error) {
```

Comment:
For whatever reason, this is still a bit awkward to me returning the name and namespace of the CatalogSource object we create. We may just want to add these fields to the deployframework object, but I'm still on the fence as to whether or not those fields are state that object needs to manage.